.travis.yml

```diff
@@ -26,6 +31,30 @@ sudo: false
 matrix:
   fast_finish: true
   exclude:
+  - node_js: "6"
```

Comment:
I've changed this

(weirdly enough one of the two allowed failures on travis works after this, the karma example)